cypress/e2e/hide_prompt_playground/provider.py

```diff
@@ -0,0 +1,52 @@
+import os
```

Comment:
i think we don't need this file. Since we are never going to open the PP we don't need providers